src/components/Dragons/Dragon.js

```diff
@@ -0,0 +1,71 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+import { useDispatch } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import { addDragonReservation, removeDragonReservation } from '../../redux/dragons/dragons';
+
+const Dragon = (props) => {
+  const dispatch = useDispatch();
+  const {
+    id, name, image, desc, reservation, type,
+  } = props;
+  // console.log(desc);
```

Comment:
- [ ] Kindly remove the commented code so your code is clean.

Reply:
Asked change implemented

Reply:
All good now 👍🏽